main.py

```diff
@@ -47,14 +54,33 @@ def fill_text_area(driver: uc_chrome, by_type: str, element_value: str, value) -
     driver = create_diver()
     go_to_page(
         driver,
-        "https://account.formula1.com/#/en/login?lead_source=web_fantasy&redirect=https%3A%2F%2Ffantasy.formula1.com%2Fapp%2F%23%2F",  # noqa: E501
+        F1_FANTASY_DRIVER_URL,
     )
     click_button_by_id(driver, By.ID, "truste-consent-button")
 
     fill_text_area(driver, By.NAME, "Login", username)
 
     elem = fill_text_area(driver, By.NAME, "Password", password)
-    elem.send_keys(Keys.RETURN)
```

Comment:
We have to keep this, or we are not going to perform the login (we are just filling the text areas).

Reply:
yep, I spent 30 minutes yesterday to understand why I didn't get anymore the token :D